### PR TITLE
[trantor] Update to 1.5.0

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]
Please add "vcpkg_fixup_pkgconfig()" in the portfile.cmake

File: ports/trantor/portfile.cmake [hunk 1]
```diff
@@ -19,6 +18,8 @@ vcpkg_install_cmake()
 # Fix CMake files
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/Trantor)
```
Comment (JonLiu1993) [inline ports/trantor/portfile.cmake:19]:
```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Thansk for your pr!

**Comment by ras0219-msft** [comment]
Thanks for the update and awesome library!

**Pull request merged**